File: src/main/java/no/nav/pensjon/regler/domain/beregning/penobjekter/ArbeidsavklaringspengerUT.java
package no.nav.pensjon.regler.domain.beregning.penobjekter;

import no.nav.pensjon.regler.domain.kode.YtelsekomponentTypeCti;

/**
 * Created by N123422 on 07.11.2014.
 */
public class ArbeidsavklaringspengerUT extends MotregningYtelseskomponent {

    private static final long serialVersionUID = -4625717954568863949L;

    public ArbeidsavklaringspengerUT() {
        super();
        ytelsekomponentType = new YtelsekomponentTypeCti("UT_AAP");
    }

    public ArbeidsavklaringspengerUT(ArbeidsavklaringspengerUT arbeidsavklaringspengerUT) {
        super(arbeidsavklaringspengerUT);
    }
}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning/penobjekter/ArbeidsavklaringspengerUT.kt
package no.nav.pensjon.regler.domain.beregning.penobjekter

import no.nav.pensjon.regler.domain.enum.YtelseskomponentTypeEnum

/**
 * Created by N123422 on 07.11.2014.
 */
class ArbeidsavklaringspengerUT : MotregningYtelseskomponent() {

    override val ytelsekomponentTypeEnum: YtelseskomponentTypeEnum = YtelseskomponentTypeEnum.UT_AAP

}